x-pack/legacy/plugins/siem/public/components/ml/anomaly/use_anomalies_table_data.ts

```diff
@@ -75,6 +79,7 @@ export const useAnomaliesTableData = ({
   const config = useContext(KibanaConfigContext);
   const capabilities = useContext(MlCapabilitiesContext);
   const userPermissions = hasMlUserPermissions(capabilities);
+  const [, dispatchToaster] = useStateToaster();
```

Comment:
nit -I am used to see it the other way around like `const dispatchToaster = useStateToaster()[1];`

Reply:
Nothing wrong with that way since it is valid ts/js, but in the all the videos for React Hooks, including the introduction of them, they use array destructoring which is the reasoning for my choice.

Let me know if I'm not seeing something with the tutorials or others that have hooks which use array indexes over destructoring as I am curious.